src/NHSD.BuyingCatalogue.Organisations.Api/Startup.cs

```diff
@@ -21,11 +22,14 @@ namespace NHSD.BuyingCatalogue.Organisations.Api
 {
     public class Startup
     {
+        private readonly IWebHostEnvironment _enviroment;
+
         private const string BearerToken = "Bearer";
 
-        public Startup(IConfiguration configuration)
+        public Startup(IConfiguration configuration, IWebHostEnvironment enviroment)
```

Comment:
Typo: `enviroment` should be `environment`.

Reply:
👍